style.css

```diff
@@ -119,4 +114,842 @@
+    font-family: "Times New Roman", Times, serif;
+    font-style: normal;
+    font-weight: 450;
+    font-size: 17px;
+    line-height: 24px;
+    color: #091e42;
+    order: 1;
+  }
+
+  .titlpost11 {
+    width: 327px;
+    height: 36px;
+    font-family: "Times New Roman", Times, serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 37px;
+    line-height: 44px;
+    display: flex;
+    align-items: center;
+    color: #091e42;
+  }
+
+  .title-work {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 0.01px;
+    position: static;
+    width: 327px;
+    height: 100px;
+    flex: none;
+  }
+
+  .h2r {
+    position: static;
+    width: 327px;
+    height: 0.01px;
+    left: 0.01px;
+    top: 107px;
+    border: 1px solid #212121;
+    flex: none;
+    order: 1;
+    margin: 24px 0.01px;
+  }
+
+  .titl {
+    position: static;
+    width: 327px;
+    height: 83px;
+    left: 0.01px;
+    top: 0.01px;
+    font-family: "Times New Roman", Times, serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 40px;
+    line-height: 52px;
+    display: flex;
+    align-items: center;
+    color: #1c1a19;
+    flex: none;
+    margin: 24px 0.01px;
+  }
+
+  .aboutdiv {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 120px 24px;
+    position: absolute;
+    width: 375px;
+    height: 563px;
+    left: 0.01px;
+    top: 4030px;
+    background-color: black;
+    margin-top: 120px;
+  }
+
+  .action2 {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 12px;
+    position: static;
+    width: 161px;
+    height: 48px;
+    left: 24px;
+    top: 395px;
+    background: #ff6b00;
+    order: 2;
+  }
+
+  .buton3 {
+    width: 161px;
+    height: 48px;
+    background: #ff6b00;
+    font-family: "Times New Roman", Times, serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 17px;
+    line-height: 24px;
+    color: #fff;
+    letter-spacing: 0.03em;
+    border: none;
+    flex: none;
+    order: 2;
+    margin: 24px 0.01px;
+  }
+
+  .descrpt {
+    position: static;
+    width: 327px;
+    height: 148px;
+    left: 24px;
+    top: 223px;
+    font-family: "Times New Roman", Times, serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    color: #f4f5f7;
+    flex: none;
+    order: 1;
+    margin: 24px 0.01px;
+  }
+
+  .prmtxt {
+    position: static;
+    width: 327px;
+    height: 79px;
+    left: 24px;
+    top: 120px;
+    font-family: "Times New Roman", Times, serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 72px;
+    line-height: 88px;
+    letter-spacing: 0.37px;
+    color: #ff6b00;
+    flex: none;
+    margin: 24px 0.01px;
+  }
+
+  .grpimg {
+    position: static;
+    width: 327px;
+    height: 237px;
+    left: 24px;
+    top: 0.01px;
+    margin-left: 187px;
+    margin-bottom: 25px;
+    order: 3;
+  }
```

Comment:
- [ ] Kindly your design is broke, because of the `width: 327px;` and `margin-left: 187px;`, kindly fix it, to match the Figma design.

![image](https://user-images.githubusercontent.com/35535130/159231348-ffa65991-1248-438b-9f98-563584353ca8.png)